sql/moz-fx-data-shared-prod/firefox_ios_derived/clients_activation_v1/checks.sql

```diff
@@ -0,0 +1,31 @@
+#fail
+{{ is_unique(["client_id"]) }}
+#fail
+{{ min_row_count(1, "`submission_date` = @submission_date") }}
+#fail
+WITH upstream_clients_count AS (
+  SELECT COUNT(*)
+  FROM `{{ project_id }}.firefox_ios.firefox_ios_clients`
+  WHERE first_seen_date = DATE_SUB(@submission_date, INTERVAL 6 DAY)
+  AND NOT is_suspicious_device_client
```

Comment:
the suspicious client filter was a funnel decision. Are we going to start applying it more broadly? Activation isn't even a funnel metric.

Reply:
Filter has been removed. Thanks!